### ISSUE TITLE
[BUG] `pip3 install taurenmd[all]` fails with zsh

### ISSUE THREAD

**Issue by Luthaf** [joaomcteixeira/taurenmd#35]
I am using zsh as my main shell, not bash. I think it tries to interpret `[` and `]` as some sort of regex, meaning I get the following error 

```
$ pip3 install taurenmd[all]
zsh: no matches found: taurenmd[all]
```

A simple fix is `pip3 install "taurenmd[all]"` (with quotes), which should also work with bash and other shells, so you may want to update the corresponding documentation.

**Comment by joaomcteixeira** [comment]
Hi @Luthaf 
Thanks for your comments. I have added your explanation to the documentation, is to be merged in #37

**Comment by joaomcteixeira** [comment]
refers to https://github.com/openjournals/joss-reviews/issues/2175
